.github/workflows/run-native-install.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9 * * *"
+
+jobs:
+  native_install:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        release: ["lilac"]
+    env:
+      OPENEDX_RELEASE: open-release/${{ matrix.release }}.master
+
+    steps:
+      - name: Remove MySQL
```

Comment:
I think that if we use something like:
```yaml
    container:
      image: ubuntu:focal 
```
It will run all the steps in a clean ubuntu docker image from Docker Hub.  So we could remove  both Remove mysql and remove mongod steps. [See](https://docs.github.com/es/actions/guides/about-service-containers#running-jobs-in-a-container).

Maybe it will be necessary to install wget, curl and sudo in order to run the Ansible Bootstrap, Create config.yml and Native Install.

I am also not sure if there could be an inconvenient in any step of the native.sh installation if it is run in a container.

Reply:
I am pretty sure some things will be failing when run inside of a container. For instance, unless I'm mistaken, the Django apps send their logs to syslog, which is not available.

Reply:
I might try it if nothing else works but so far the other services are installing just fine (see [post](https://discuss.openedx.org/t/help-running-native-install-inside-a-github-runner-for-ci/4855)).